src/cr/cube/cube.py

```diff
@@ -459,6 +486,32 @@ def _measures(self):
         """
         return _Measures(self._cube_dict, self._all_dimensions)
 
+    @lazyproperty
+    def _numeric_array_dimension(self):
+        """Column dimensions object inflated according to the mean subvariables."""
+        column_dimension = {
+            "references": {"alias": "mean", "name": "mean"},
+            "type": {"elements": [], "class": "enum", "subtype": {"class": "variable"}},
+        }
+        subrefs = self._mean_subreferences
+        # ---cube inflation---
```

Comment:
This is not yet inflation. It happens in the original `dict` code, and this comment should probably go there. We still need some commenting here, but here, try to explain _what_ we're building, rather than _what_ we need to _do_ with it. And then explain what we need to do, in the place we're actually doing it.